### ISSUE TITLE
Add a model zoo of sorts?

### ISSUE THREAD

**Issue by reiinakano** [tensorflow/tfjs-core#131]
I've been looking to use some pretrained VGG models in deeplearnjs but it seems there's no `deeplearnjs/models` model repository yet? Is this something you have on your roadmap? What do you think is the best way to use pretrained VGG models from TF in deeplearnjs?

**Comment by nsthorat** [comment]
We have a "models" directory in `demos/models`, but it's not what we're going to converge around. Right now we have a pretrained SqueezeNet (~5MB, AlexNet level accuracy on ImageNet) which you can use out of the box for transfer learning tasks.

https://github.com/PAIR-code/deeplearnjs/blob/master/demos/imagenet/imagenet-demo.ts#L117

What do you want to do with the imagenet model?

**Comment by reiinakano** [comment]
Was planning to do neural style transfer as an exercise :)

Although I may just go for fast neural style instead so that no training needs to be done on the browser which may take a while. Although regular style transfer from Gatys would let users use arbitrary styles..

**Comment by nsthorat** [comment]
You should be able to use SqueezeNet activations as feature maps for neural style transfer, in the 
 same way you'd use VGG activations, right? (not fast neural, the one where you pose it as an optimization problem between activations)

The SqueezeNet link above gives you activations at every layer, this is how we we render activations in this demo: https://pair-code.github.io/deeplearnjs/demos/imagenet/imagenet-demo.html

**Comment by reiinakano** [comment]
I will try this, although I'm not sure how well SqueezeNet will work for the task.

By the way, for doing Fast Neural Style Transfer, is there any way to achieve the following in DeeplearnJS? 
* Instance normalization
* Transpose conv layer

Sorry, couldn't see on the API if those were available.

**Comment by andrewlook** [comment]
@nsthorat FYI I think the demo link you posted is now broken (i see an S3 error). This one works though: https://deeplearnjs.org/demos/imagenet/

**Comment by nsthorat** [comment]
We're going to think about this with TensorFlow.js. No official plans yet, but this is being discussed. Going to close this off as we'll start a larger discussion down the road.
